affiliates/base/static/css/base.styl

```diff
@@ -836,13 +836,44 @@ body.noscroll {
     }
 }
 
-#smartbanners {
+#smartbanners, #friends-msg {
     display: none;
 }
 
+.friends-msg-modal {
+    background: transparent url(/static/img/affiliates-to-friends.png) top center no-repeat;
+    padding-top: 145px;
+    h4 {
+        text-align: center;
+    }
+    p {
+        text-align: center;
```

Comment:
Nitpick: Could combine these rules, or even set text-align on the whole container unless that interferes with other content.